anaconda.spec.in

```diff
@@ -195,6 +195,39 @@ The anaconda-install-env-deps metapackage lists all installation environment dep
 This makes it possible for packages (such as Initial Setup) to depend on the main Anaconda package without
 pulling in all the install time dependencies as well.
 
+%package install-image-deps
+Summary: Installation image specific dependencies
+Requires: udisks2-iscsi
+Requires: libblockdev-plugins-all >= %{libblockdevver}
+# active directory/freeipa join support
+Requires: realmd
+Requires: isomd5sum >= %{isomd5sumver}
+%ifarch %{ix86} x86_64
+Requires: fcoe-utils >= %{fcoeutilsver}
+%endif
+# likely HFS+ resize support
+%ifarch %{ix86} x86_64
+%if ! 0%{?rhel}
+Requires: hfsplus-tools
+%endif
+%endif
+# kexec support
+Requires: kexec-tools
+Requires: createrepo_c
+# run's on TTY1 in install env
+Requires: tmux
+# install time crash handling
+Requires: gdb
+Requires: rsync
+Requires: device-mapper-multipath
+Requires: kpartx
+Requires: nm-connection-editor
+Recommends: zram-generator-defaults
+
+%description install-image-deps
+The anaconda-install-image-deps metapackage lists all installation image dependencies.
+Add this package to an image build to ensure all Anaconda capabilities are supported.
```

Comment:
I would like to see here the `boot.iso` word to make it clear for people who are familiar with that terminology.

Reply:
I would include somewhere `lorax` into this sentence as an example of the tool that would use this package.